barrier3d/barrier3d.py

```diff
@@ -96,80 +97,89 @@ def Shrubs(
 
         ShrubDomainAll = ShrubDomainFemale + ShrubDomainMale
 
-        BeachWidth = 60 / 10  # Temp
-
-        # ### Burial
-        # Find heights of shrub plants
-        ShrubHeight = ShrubDomainAll * self._MaxShrubHeight
-
-        # Kill all shrubs buried by over depth limit or eroded
-        ShrubDomainDead[
-            BurialDomain > (self._BurialLimit * ShrubHeight)
-        ] = ShrubDomainAll[
-            BurialDomain > (self._BurialLimit * ShrubHeight)
-        ]  # Transfer to dead domain
-        ShrubDomainDead[
-            BurialDomain >= 0.3
-        ] = 0  # If buried by more than 3 m, considered shrub to be gone(dead or alive)
-        ShrubDomainDead[BurialDomain < self._UprootLimit] = ShrubDomainAll[
-            BurialDomain < self._UprootLimit
-        ]  # Transfer to dead domain
-
-        ShrubDomainFemale[BurialDomain > (self._BurialLimit * ShrubHeight)] = 0
-        ShrubDomainFemale[BurialDomain < self._UprootLimit] = 0
-        ShrubDomainMale[BurialDomain > (self._BurialLimit * ShrubHeight)] = 0
-        ShrubDomainMale[BurialDomain < self._UprootLimit] = 0
-
-        BurialDomain[
-            BurialDomain > (self._BurialLimit * ShrubHeight)
-        ] = 0  # Reset burial domain
-        BurialDomain[BurialDomain < self._UprootLimit] = 0  # Reset burial domain
-        BurialDomain[BurialDomain >= 0.3] = 0  # Reset burial domain
+        BeachWidth = 40 / 10  # Calculated based on berm (dune toe) elevation of 1.44 MSL, and beach slope of 0.04 (i.e., 1.44/0.04 = 47.5 ~= 40 m)
```

Comment:
This should be a class variable since it is based on the beach slope, which is specified in the input file as "beta"

Reply:
```suggestion
        BeachWidth = self._bermEl / self._beta   # Calculated based on berm (dune toe) elevation of 1.44 MSL, and beach slope of 0.04 (i.e., 1.44/0.04 = 47.5 ~= 40 m)
```

Reply:
I was suggesting something like this. @irbreeves, what do you think? Does this work?